Doc tests fix for llm transformations (#400)
agentgateway/website: assets/agw-docs/pages/agentgateway/llm/transformations.md

EOF
{{< /doc-test >}}

> 1. Create an {{< reuse "agw-docs/snippets/trafficpolicy.md" >}} resource to apply an LLM request transformation. The following example limits `max_completion_tokens` to no more than 10 by taking the smaller of the client's requested value and 10, regardless of what the client requests.

Reviewer: ```suggestion
1. Create an {{< reuse "agw-docs/snippets/trafficpolicy.md" >}} resource to apply an LLM request transformation. The following example limits `max_completion_tokens` to no more than 10. If the client requests less than 10 tokens, this number is applied. If the client requests more than 10 tokens, the maximum number of 10 is applied.  
```

   x-actual-model: gpt-4o-mini
   ```

   {{< callout type="info" >}}

>    When sending traffic to the gateway with traffic compression enabled, such as gzip or br,  the CEL expression could fail. If a header is missing from a response, try a different accept encoding in your request.

Reviewer: ```suggestion
   When sending traffic to the gateway with traffic compression enabled, such as `gzip` or `br`, the CEL expression could fail. If a header is missing from a response, try a different `accept-encoding` header in your request.
```

* `llm.requestModel`: The model name from the original request.
* `llm.responseModel`: The model name the upstream LLM provider reported in the response.

> Use [`metadata`]({{< link-hextra path="/traffic-management/transformations/templating-language/#pre-compute-values-with-metadata" >}}) to pre-compute the `llm` context values, and `default()` in the `set` expressions to fall back to parsing the raw body if the variable is unavailable. This approach computes each value once and keeps the `x-model-fallback` comparison readable:

Reviewer: ```suggestion
Use the [`metadata`]({{< link-hextra path="/traffic-management/transformations/templating-language/#pre-compute-values-with-metadata" >}}) context variable to pre-compute LLM model data, and the `default()` function in the `set` expressions to fall back to parsing the raw body if the metadata context variable is unavailable. This approach computes each value once and keeps the `x-model-fallback` comparison readable:
```
